cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-upgrading.adoc

```diff
@@ -0,0 +1,78 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-getting-started-upgrading"]
+= Tutorial: Upgrading your cluster
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-getting-started-upgrading
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2024-01-08
+
+{product-title} (ROSA) executes all cluster updates as part of the managed service. You do not need to run any commands or make changes to the cluster. You can schedule the upgrades at a convenient time.
+
+Ways to schedule a cluster upgrade include:
+
+* *Manually using the command line interface (CLI)* - Start a one-time immediate upgrade or schedule a one-time upgrade for a future date and time.
```

Comment:
Instead of a dash for these bulleted items, maybe a colon...or something else? :)